register_and_run.sh

```diff
@@ -1,23 +1,19 @@
-#!/bin/bash -ex
+#!/bin/bash -e
 
 error() { echo "$@" >&2 ; exit 1 ; }
 
 # Check for some required environment variables
 [[ -n "${GITLABURL}" ]] || error "GITLABURL environment variable must be set"
-[[ -n "${RUNNERTOKEN}" ]] || error "RUNNERTOKEN environment variable must be set"
+[[ -n "${TOKEN}" ]] || error "TOKEN environment variable must be set"
 
 # Set some sensible default values
 : "${EXECUTOR:=docker}"
 : "${DOCKERIMAGE:=alpine:latest}"
-: "${RUNNERNAME:=gitlab runner}"
-: "${TAGLIST:=docker,auto}"
 
 echo "GITLABURL: ${GITLABURL}"
-echo "RUNNERTOKEN: ${RUNNERTOKEN}"
+echo "TOKEN: [redacted]"
```

Comment:
Good call on the redaction.... but could this  be [redacted] if the token is set or [empty] if the token has not been set? The latter strikes me as a likely error case that it would be good to have some debugging help with

Reply:
A few lines above this, an error is thrown if $TOKEN is unset or empty, so this guard is already taken care of.